core/src/com/unciv/ui/cityscreen/ConstructionInfoTable.kt

```diff
@@ -78,11 +89,36 @@ class ConstructionInfoTable(val cityScreen: CityScreen): Table() {
             descriptionLabel.wrap = true
             add(descriptionLabel).colspan(2).width(stage.width / 4)
 
-            val link = (construction as? IRulesetObject)?.makeLink() ?: return
-            if (link.isEmpty()) return
-            touchable = Touchable.enabled
-            onClick {
-                UncivGame.Current.pushScreen(CivilopediaScreen(city.getRuleset(), link = link))
+            // Show sell button if construction is a currently sellable building
+            if (construction is Building && cityConstructions.isBuilt(construction.name)
+                    && construction.isSellable()) {
+                val sellAmount = cityScreen.city.getGoldForSellingBuilding(construction.name)
+                val sellText = "Sell [$sellAmount] " + Fonts.gold
```

Comment:
`Sell` This line is not translated in other languages.

![sell](https://user-images.githubusercontent.com/11946570/208314290-0f3ed4d7-7e2c-49ea-9395-a0858fe30abf.png)